Note.cpp

```diff
@@ -6,23 +6,15 @@
 
 #include "Note.h"
 
-Note::Note()
+Note::Note(int str, int fret, Finger digit, char note) : stringNum(str), fretNum(fret), finger(digit), key(note)
 {
-    stringNum = 1;
-    fretNum = 0;
-    finger = none;
-    //std::cout << "Note default CTOR" << std::endl; 
+    //std::cout << "Note CTOR" << std::endl;
 }
 
-Note::Note(int str, int fret, Finger digit, char note)
+Note::Note()
 {
-    stringNum = str;
-    fretNum = fret;
-    finger = digit;
-
-    key = note;
-
-    //std::cout << "Note CTOR" << std::endl;
+    Note(1,0,none,'E');
```

Comment:
This is constructing and destructing an anonymous `Note` all in one line.  It is not initializing the `Note` instance being created on `line 14` with specific values.
Use the member initializer list to call the constructor that takes arguments:
```
Note::Note() : Note( /* args */ ) { } 
```
- [ ] revise

Reply:
ok done